ElastosBountyProgram/front-end/src/module/page/profile/tasks/Container.js

```diff
@@ -83,7 +83,9 @@ export default createContainer(Component, (state) => {
          */
         async getTasks(currentUserId) {
             return taskService.index({
-                profileListFor: currentUserId
+                profileListFor: currentUserId,
+                type: TASK_TYPE.TASK,
```

Comment:
Also TASK_TYPE.EVENT (might need backend adjustment to support arrays here - if not there already)

Reply:
Fixed. Backend changes were not necessary.

Reply:
Are you sure? Doesnt seem we support arrays for types on the backend.
Consider task/list.ts
```
if (param.type && _.values(constant.TASK_TYPE).includes(param.type)) {
            query.type = param.type;
        }
```

includes alone will fail, not even going to mongo yet.